tests/testPhotoCal.py

```diff
@@ -161,9 +194,12 @@ def testFlags(self):
             if source.get("calib_photometryUsed"):
                 used += 1
             if source.get("calib_photometryReserved"):
+                reservedSum -= source.getId()
                 reserved += 1
-        self.assertEqual(reserved, int(.3 * candidates))
+        self.assertEqual(reserved, int(.3 * (candidates + reserved)))
         self.assertLessEqual(used, (candidates - reserved))
+        #   this sum will not generally be 0 if the set of reserved candidates is different
+        self.assertNotEqual(reservedSum, 0)
```

Comment:
Yes, I agree, although at the time it seemed adequate for a unit test where the expId's were fixed.  However, I agree it is a little sloppy.